src/Commands/Make/ClassCommand.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Nwidart\Modules\Commands\Make;
+
+use Illuminate\Support\Str;
+use Nwidart\Modules\Support\Config\GenerateConfigReader;
+use Nwidart\Modules\Support\Stub;
+use Nwidart\Modules\Traits\ModuleCommandTrait;
+
+class ClassCommand extends GeneratorCommand
+{
+    use ModuleCommandTrait;
+
+    /**
+     * The name and signature of the console command.
+     */
+    protected $signature = 'module:make-class
+        {--t|type=class : The type of class, e.g. class, service, repository, contract, etc.}
+        {--p|plain : Create the class without suffix (type)}
+        {--i|invokable : Generate a single method, invokable class}
```

Comment:
Invokable option not yet implemented.

Reply:
I agree with you. I had planned to implement it later on.